### PR TITLE
[IMP] web: fix dropdown-item checkmark position

### PR DESCRIPTION
**Pull request by xlu-odoo**
Prior to this commit, the positioning of the checkmark in the project task state dropdown was slightly off, not optimally aligned.

This commit solves this alignment.

<table>
<tr>
 <td>Before</td>
 <td> After</td>
<tr>
 <td>
 before
</td>
 <td>after
 </td>
</table>

task-3575446

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/webclient/webclient.scss [hunk 1]
```diff
@@ -72,7 +72,7 @@ kbd {
     &:not(.dropdown-item_active_noarrow) {
       &:before {
         @include o-position-absolute(0);
-        transform: translate(-1.5em, 90%);
+        transform: translate(-1em, 75%);
```
[the remark this answers is not in the document]
Comment (xlu-odoo) [reply addons/web/static/src/webclient/webclient.scss]: Good idea imo. Updated 👍 thanks

**Review by chgo-odoo — APPROVED** [review]
Thanks @xlu-odoo LGTM, you can remove the draft state  👍

**Comment by xlu-odoo** [comment]
@pparidans can you look at this PR and eventually merge? Thanks 🙏

**Comment by Brieuc-brd** [comment]
Hey @xlu-odoo 👋 

I just quickly checked your PR, and I think it should target `saas-16.3` since this issue appears from this version.

Could you check?

**Comment by xlu-odoo** [comment]
> Hey @xlu-odoo 👋
> 
> I just quickly checked your PR, and I think it should target `saas-16.3` since this issue appears from this version.
> 
> Could you check?

Thanks @Brieuc-brd , retargeted it

**Review by pparidans — CHANGES_REQUESTED** [review]
Well actually it breaks the SearchPanel's Filter/Group by selected state :
image

**Comment by Brieuc-brd** [comment]
Thanks for the review @pparidans.
@xlu-odoo is off for the moment, I updated the branch to change the margin of the dropdown item instead of the main scss. Could you take a look?

**Comment by Brieuc-brd** [comment]
@stefanorigano 👋 
Little reminder :)

**Review by stefanorigano — COMMENTED** [review]
Hello @xlu-odoo , thanks.
Could you rebase your branch, runbot is red,
Thanks again

**Comment by xlu-odoo** [comment]
> Hello @xlu-odoo , thanks. Could you rebase your branch, runbot is red, Thanks again

Rebased (and green) @stefanorigano  :)

**Review by stefanorigano — APPROVED** [review]
Thanks @xlu-odoo , LGTM 👍 

@pparidans , could you have a look? thanks

**Pull request closed without merging**